package.json

```diff
@@ -11,7 +11,7 @@
     "archiver": "^0.14.4",
     "bluebird": "^3.4.7",
     "chalk": "^1.1.1",
-    "devkit-spriter": "git+https://github.com/gameclosure/devkit-spriter.git#es6",
+    "devkit-spriter": "git+https://github.com/bjornstar/devkit-spriter.git#serial-load",
```

Comment:
once https://github.com/gameclosure/devkit-spriter/pull/3 lands I can switch this back.